### DISCUSSION TITLE
Is writing of labels implemented in the new version?

### DISCUSSION THREAD

**Discussion by pgrond** [LeMyst/WikibaseIntegrator#239] [Q&A] [answered]
I see a new Label class, but that only has a pass. Is it possible to write a label? If so, how should this be done? If not, what would be needed to implement this?

**Comment by LeMyst** [comment, marked as answer]
Hello @pgrond ,
Sorry for the late reply.

yes, label is implemented in v0.12:

```python
wbi = WikibaseIntegrator()
item = wbi.item.get(entity_id='Q141806')
item.labels.set(language='en', value='An updated item', action_if_exists=ActionIfExists.KEEP)
item.write()
```
